### ISSUE TITLE
Validation Data Used as Test??

### ISSUE THREAD

**Issue by guglielmocamporese** [BraveGump/CGDL-for-Open-Set-Recognition#9]
Is it fair to evaluate the openset task with the validation data from the closed classification task? Wouldn't be fair to do something like:
1. split MNIST train into train and validation,
2. train on closed classification on train and validate on validation, 
3. use the MNIST test as test on openset classification, not as validation.

In this way, the test is not seen on the closed classification task.

Best,

Guglielmo

**Comment by BraveGump** [comment]
> Is it fair to evaluate the openset task with the validation data from the closed classification task? Wouldn't be fair to do something like:
> 
> 1. split MNIST train into train and validation,
> 2. train on closed classification on train and validate on validation,
> 3. use the MNIST test as test on openset classification, not as validation.
> 
> In this way, the test is not seen on the closed classification task.
> 
> Best,
> 
> Guglielmo

Certainly, your suggestion is recommendable and looks more reasonable.
